### ISSUE TITLE
Roblox crashing while using Bloxstrap. No error code

### ISSUE THREAD

**Issue by CrawLSold** [pizzaboxer/bloxstrap#2561]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [ ] My answer given in the checkbox above is a lie.

### What problem did you encounter?

Roblox randomly crashes while using bloxstrap. No error code.
I've only encountered this issue while using Bloxstrap. No special fastflags were used other than 1-21 graphic mode. 
I also did reinstall bloxstrap after this issue. Problem was not solved. Though, problem seems to be solved while I'm launching my roblox with no special fastflags (without 1-21 graphic mode).

Sounds like a bloxstrap issue, what do you say?

Adding; Roblox does not crash nor gives me an error code. Closes randomly, for no reason. Tested on other variants of games aswell, same thing happened.

**Comment by pizzaboxer** [comment]
So just to clarify. using Bloxstrap shows a crash dialog while using Roblox just unexpectedly closes?

**Comment by CrawLSold** [comment]
> So just to clarify. using Bloxstrap shows a crash dialog while using Roblox just unexpectedly closes?

Disregard the situation above, I've found my solution. But either way;

No, bloxstrap was not showing any dialog at all. Instant crash and the game is gone.
I've fixed the situation by unplugging my game console from my computer. Aswell as making sure all my drivers were up to date.

After all these, I can finally use bloxstrap with no issues. Tho, whenever I plug my console back on- I still face the same issue. Checked with other consoles. It happens on every single one of them.
